.github/workflows/tests.yml

```diff
@@ -178,7 +189,18 @@ jobs:
           python setup.py build_ext -i
           python setup.py bdist_wheel
 
+      - name: Install zodbpickle and dependencies (3.13.0-alpha - 3.13.0)
+        if: matrix.python-version == '3.13.0-alpha - 3.13.0'
+        run: |
+          # Install to collect dependencies into the (pip) cache.
+          # cffi will probably have no public release until a Python 3.13 beta
+          # or even RC release, see https://github.com/python-cffi/cffi/issues/23
+          echo "cffi @ git+https://github.com/python-cffi/cffi.git@954cab4f889fb019a7f90df153ee1be501495f58" > cffi_constraint.txt
+          # Use "--pre" here because dependencies with support for this future
+          # Python release may only be available as pre-releases
+          PIP_CONSTRAINT=$PWD/cffi_constraint.txt pip install --pre .[test]
```

Comment:
I'm not sure why we jump through hoops here to install `cffi`:  AFAICT, this is the only file inside `zodbpickle` which mentions it.  Is this from the `meta` repo?

Reply:
This `cffi` dance actually came from the `meta` repo. It is not needed here and it will get removed when updating with the current version of `meta/config`: I removed it over there because it did not work as expected.

Thank you very much for fixing this issue.